src/angular/notification/notification.scss

```diff
@@ -47,20 +47,27 @@
 }
 
 :where(html.sbb-lean) .sbb-notification-success {
-  color: var(--sbb-color-green);
-  background-color: var(--sbb-color-background);
-  border-color: var(--sbb-color-green);
+  color: var(--sbb-notification-label-color-success);
+  background-color: var(--sbb-notification-background-color-success);
+  border-color: var(--sbb-notification-border-color-success);
 }
 
 :where(html.sbb-lean) .sbb-notification-info {
```

Comment:
Please check whether these `:where(html.sbb-lean)` blocks are still needed.

Reply:
without the ":where(html.sbb-lean)" the Notifications in the Standard Mode change.

Reply:
It's probably because the default colors of the variables are not set correctly.
You need to initialize the variables with the colors for standard mode, override it with the ones for lean mode in the `:where(html.sbb-lean)` block and the ones for the dark mode in the `:where(html.sbb-lean.sbb-dark)` block. Or am I missing something?